support/ebpf/dotnet_tracer.ebpf.c

```diff
@@ -102,12 +102,31 @@ dotnet_find_code_start(PerCPURecord *record, DotnetProcInfo *vi, u64 pc, u64 *co
     // This is unrolled several times, so it needs to be minimal in size.
     // And currently this is the major limit for DOTNET_FRAMES_PER_PROGRAM.
     int orig_pos = pos;
+    if (val == 0) {
+      // pos is fixed to map_elements-2 at this point, and is even.
+      // convert it from u32 to u64 offset
+      int pos64 = (map_elements - 2) / 2 - 1;
+      u64 val64 = scratch->map64[pos64];
```

Comment:
seems this earlier change got dropped. was this intentional?
```suggestion
      int pos64 = (map_elements - 2) / 2 ;
      u64 val64 = scratch->map64[--pos64];
```